common/app/views/fragments/email/stylesheets/front.scala.html

```diff
@@ -45,14 +45,17 @@
         text-decoration: none;
     }
     .facia-card__text {
-        padding: 0 10px 26px 10px !important;
+        padding: 0 10px 0px 10px !important;
```

Comment:
That sounds promising! Might be best just to add a comment saying this on the top of the file? What do you think?